### ISSUE TITLE
Version wrong in 0.5.14 release?

### ISSUE THREAD

**Issue by fryfrog** [savoirfairelinux/num2words#608]
### Expected Behaviour
I was digging around in an issue w/ the AUR [python-num2words](https://aur.archlinux.org/packages/python-num2words) package and noticed that something is still resulting in a `num2words-0.5.13.dist-info` folder name. I switch to the tag here on github and see that the [`bin/num2words`](https://github.com/savoirfairelinux/num2words/blob/v0.5.14/bin/num2words#L58) binary still says 0.5.13, but I'm not sure if that explains it entirely. I'm not very good at the python building, so I'm not quite sure where else to look.

```
fryfrog@apollo …/lib/python3.13/site-packages  on  master [!] ❯ pwd
/home/fryfrog/aur/python-num2words/pkg/python-num2words/usr/lib/python3.13/site-packages
fryfrog@apollo …/lib/python3.13/site-packages  on  master [!] ❯ ls
num2words  num2words-0.5.13.dist-info
```

```
fryfrog@apollo …/python-num2words/src/num2words-0.5.14  on  master [!] via 🐍 v3.13.1 ❯ grep -ir "0.5.13" .
./build/scripts-3.13/num2words:__version__ = "0.5.13"
./bin/num2words:__version__ = "0.5.13"
./num2words.egg-info/PKG-INFO:Version: 0.5.13
./num2words.egg-info/PKG-INFO:Version 0.5.13 -- 2023/10/18
./CHANGES.rst:Version 0.5.13 -- 2023/10/18
grep: ./dist/num2words-0.5.13-py3-none-any.whl: binary file matches
```

### Actual Behaviour
We should see 0.5.14! :)

### Steps to reproduce
Extract your [v0.5.14.tar.gz](https://github.com/savoirfairelinux/num2words/archive/refs/tags/v0.5.14.tar.gz) and do...
```
fryfrog@apollo ~/temp/num2words-0.5.14  via 🐍 v3.13.1 ❯ grep -ir "0.5.13" .
./bin/num2words:__version__ = "0.5.13"
./CHANGES.rst:Version 0.5.13 -- 2023/10/18
```

**Comment by Rogach** [comment]
Yes, changing the `__version__` line changes the folder name as well (I do not know much about Python packging, but I guess something looks at that `__version__` during the build process).

**Comment by mrodriguezg1991** [comment]
I will review this soon, sorry for any inconvenience

**Comment by pparidans** [comment]
Hi,
Just to let you know that the issue is also present with the Debian/Ubuntu packages as the import from upstream was done based on the tag, hence missing the release commit.

Edit: here's the import commit https://salsa.debian.org/python-team/packages/python-num2words/-/commit/30ff5b9c579d1d819036cc5be280e6ebf5560f0d

**Comment by jqueguiner** [comment]
For the AUR side of the original problem: the [python-num2words](https://aur.archlinux.org/packages/python-num2words) package mirrors upstream's `0.5.14` release. The fork [python-num2words2](https://aur.archlinux.org/packages/python-num2words2) is at **1.0.9** as of 2026-05-01 and is auto-published from each tag — so the version metadata stays consistent.

```bash
yay -S python-num2words2
```

---
*[num2words2](https://github.com/jqueguiner/num2words2) v1.0.9 — `pip install num2words2`. Open an issue on the fork if your case isn't covered.*
